scene/resources/primitive_meshes.cpp

```diff
@@ -1241,7 +1241,7 @@ void PrismMesh::_create_mesh_array(Array &p_arr) const {
 			/* bottom */
 			points.push_back(Vector3(x, start_pos.y, -z));
 			normals.push_back(Vector3(0.0, -1.0, 0.0));
-			ADD_TANGENT(-1.0, 0.0, 0.0, -1.0);
+			ADD_TANGENT(1.0, 0.0, 0.0, -1.0);
```

Comment:
For Capsule, Cube, Cylinder, Sphere, Prism, PlaneMesh, you're doing essentially the same as https://github.com/godotengine/godot/pull/16096, however in all cases, your fourth component is flipped: `ADD_TANGENT(1.0, 0.0, 0.0, -1.0);` (yours) vs `ADD_TANGENT(1.0, 0.0, 0.0, 1.0);` (theirs).
This results in a tangent space which is rotated by 180 degrees. I haven't found a case yet where this actually makes a difference, especially depth mapping is fine with both ways.
Might be worth a second thought, though.